docs/languages/python.md

```diff
@@ -0,0 +1,83 @@
+
+Quickly see methods, class members, and documentation.
+
+<video id="python-code-completion-video" src="/images/python_python-intellisense.mp4" poster="/images/python_python-intellisense-placeholder.png" autoplay loop controls muted></video>
+
+> Tip: trigger code completion with `kb(editor.action.triggerSuggest)`.
+
+## Linting
+
+Linting is the analysis of your Python code for potential errors. Use Visual Studio Code to quickly navigate to the errors and warnings in your code. 
+
+<video id="python-linting-video" src="/images/python_python-linting.mp4" poster="/images/python_python-linting-placeholder.png" autoplay loop controls muted></video>
+
+> Tip: [Don Jayamanne's Python extension](https://marketplace.visualstudio.com/items?itemName=donjayamanne.python) gives you the option of using three different linters - [Pylint](https://www.pylint.org/), [Pep8](https://pypi.python.org/pypi/pep8), and [Flake8](https://flake8.readthedocs.io/en/latest/). See the [wiki](https://github.com/DonJayamanne/pythonVSCode/wiki/Linting) for more details. 
+
+## Debugging
+
+No more `print` statement debugging! Set breakpoints, watch variables, inspect data, and use the debug console. Debug a number of different type of Python applications, including multi threaded, web, and remote applications. 
+
+<video id="python-debugging-video" src="/images/python_python-debugging.mp4" poster="/images/python_python-debugging-placeholder.png" autoplay loop controls muted></video>
+
+> Tip: Follow the instructions in the [wiki](https://github.com/DonJayamanne/pythonVSCode/wiki/Debugging) for getting started with debugging, including setting up your `launch.json` and troubleshooting common issues. 
+
+> Tip: Read more about general information about debugging in Visual Studio Code in the [debugging document](/docs/editor/debugging). 
+
+## Snippets
+
+Snippets will take productivity to the next level. You can configure [your own snippets](/docs/customization/userdefinedsnippets) and use snippets provided by an extension. 
+
+<video id="python-snippets-video" src="/images/python_python-snippets.mp4" poster="/images/python_python-snippets-placeholder.png" autoplay loop controls muted></video>
+
+> Tip: Snippets appear in the same way as code completion - `kb(editor.action.triggerSuggest)`. 
+
+## Configuration
+
+You will need an [extension](#_install-python-extension) and [Python](https://www.python.org/downloads/) installed. Other dependencies are optional and depend on the features you want to use. Read through the requirements on the [extension's README](https://marketplace.visualstudio.com/items?itemName=donjayamanne.python#requirements). 
+
+## Next Steps
+
+* [Install an Extension](#_install-python-extension) - Python extension are available in the [Marketplace](https://marketplace.visualstudio.com/vscode).
```

Comment:
Do you want a link to /docs/editor/extension-gallery.md if people are new to extensions?  That topic describes extension, the Marketplace, how to install/update/uninstall.
Next steps usually links to another topic not back to an earlier header in the current topic.